### PR TITLE
2873: Set fixed timezone for jest

### PR DESCRIPTION
**Pull request by bahaaTuffaha**
### Short description

Timestamps at DatabaseConnector causing failure in Tests when running from a different time zone (my case is GMT+3).

### Proposed changes

- Added this line: `process.env.TZ = 'Europe/Berlin'` to jest.config.ts at native and web.

### Side effects

None

### Resolved issues

Fixes: [#2873](https://github.com/digitalfabrik/integreat-app/issues/2873)

---

### CODE REVIEW

**Review by steffenkleinle — APPROVED** [review]

**Review by LeandraH — APPROVED** [review]
Nicely done, thank you!

**Comment by LeandraH** [comment]
@bahaaTuffaha You can merge your PRs once they have the required amount of approvals, like this one  🎉

**Pull request merged**
